packages/theme-patternfly-org/pages/red-hat-font.md

```diff
@@ -3,96 +3,35 @@ id: Red Hat font
+For example, `--pf-global--FontFamily--redhatfont--sans-serif`.
 
-- "RedHatText": uses a smaller/lighter character with hinting that enables it to fit the pixel grid well at smaller sizes like body text.
-
-- `--pf-global--FontFamily--redhatfont--sans-serif`
-- Font weights
+**Font weights**
 
 - 400 (normal)
 - 700 (bold)
 
-- "RedHatDisplay": uses a wider/heavier character which matches our logo and is designed to be used for headings and other large sizes.
+### Red Hat Display
+
+Red Hat Display font uses broader and heavier characters which matches the Red Hat logo. Use the Red Hat Display font for headings and other large text sizes.
 
-- `--pf-global--FontFamily--redhatfont--heading--sans-serif`
-- Font weights
+For example, `--pf-global--FontFamily--redhatfont--heading--sans-serif`.
+
+**Font weights**
 
 - 300 (light - rarely used)
 - 400 (normal)
 - 700 (bold)
 
-- Adds new global variable for use with RedHatDisplay
-
-- `--pf-global--FontFamily--heading--sans-serif`
-
-- No longer using a semi-bold font weight with RedHatDisplay/RedHatText
-- Adds new variable for "bold" for use with RedHatDisplay/RedHatText since the bold value was 600 with Overpass, and is now 700 with RedHatDisplay/RedHatText
-
-- `--pf-global--FontWeight--redhatfont--bold`
-
-- Changes the value for `--pf-global--FontSize--xl` from 21px to 20px
-
-- Works better on a mathematical scale for relative sizing
-
-## Red Hat font changes
-
-### Global variables
-
-- SansSerif font var changes from Overpass to RedHatText
-- Heading font var defined as RedHatDisplay
-- Bold font var changes from 600 to 700
-- Semi-bold font var maps to the bold font var
-- Link font-weight changes from semi-bold to normal
-
-### Components
-
-**Alert**:
-
-- Title now defined as bold
-
-**Breadcrumb**:
-
-- Link and item font-weight changes from semi-bold to normal
-
-**Button**:
-
-- Font-weight changes from semi-bold to normal
-
-**Card**:
-
-- Card header font-weight is defined as bold
-- Card header font-family is defined as RedHatText
-
-**Content**:
-
-- `<h1>` - `<h6>` use RedHatDisplay
-- `<h2>` line-height changes from `--pf-global--LineHeight--sm` (1.3) to `--pf-global--LineHeight--md` (1.5)
-- `<blockquote>` font-weight changes from light to normal
-- `<h4>`, `<h5>`, and `<h6>` font-weight changes from semi-bold to normal
-
-**Expandable**:
-
-- Toggle font-weight changes from semi-bold to normal
-
-**Empty state**:
-
-- Title font-size is now defined as `--pf-global--FontSize--xl` (20px)
-
-**Title**:
-
-- Font-family changes to RedHatDisplay
-- The large (`.pf-m-lg`) and medium (`.pf-m-md`) variations’ font-weight changes from semi-bold to normal
-
-
 ### Existing overrides
```

Comment:
I'm not sure about this section, does it need to be here? is it something users should know when using Red Hat font?
Otherwise we should remove this as well.